app/processing/statement/components/get-detailed-funding.js

```diff
@@ -0,0 +1,28 @@
+const getFundings = require('../../funding')
+const getInvoiceLine = require('../../invoice-line')
+const { convertToPounds } = require('../../../utility')
+
+const getDetailedFunding = async (calculationId, paymentRequestId) => {

```

Comment:
Need to create a mapping function file

Also every returned value needs to be string

Reply:
As above.